internal/provider/resource_gitlab_project_approval_rule.go

```diff
@@ -43,6 +49,13 @@ var _ = registerResource("gitlab_project_approval_rule", func() *schema.Resource
 				Type:        schema.TypeInt,
 				Required:    true,
 			},
+			"rule_type": {
+				Description:      fmt.Sprintf("String, defaults to 'regular'. The type of rule. `any_approver` is a pre-configured default rule with `approvals_required` at `0`. Valid values are %s.", renderValueListForDocs(validRuleTypeValues)),
+				Type:             schema.TypeString,
+				Optional:         true,
+				Default:          "regular",
```

Comment:
NIT:

```suggestion
```

I'd leave it up to GitLab to decide on the `rule_type` ... thus, we don't send it, if it wasn't set :)

Reply:
Ah, right. makes sense :)